source/django-get-started/django-create-app.txt

```diff
@@ -0,0 +1,255 @@
+.. _django-get-started-create-app:
+
+=====================
+Create an Application
+=====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: app, odm, code example
+
+In your ``quickstart`` project, you can create an application
+that interacts with the Atlas sample database called ``sample_mflix``.
+This database contains a ``movies`` collection, which stores
+information about movies. The database also contains a ``users``
+collection, which stores information about movie viewers who use 
+a streaming service.
+
+To learn more about the ``sample_mflix`` database, see :atlas:`Sample Mflix Dataset
+</sample-data/sample-mflix/>` in the Atlas documentation.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a "sample_mflix" app
+
+      From your project's root directory, run the following command to create a 
+      new Django app called ``sample_mflix`` based on a custom template:
+      
+      .. code-block:: bash
+
+         python3 manage.py startapp sample_mflix --template https://github.com/mongodb-labs/django-mongodb-app/archive/refs/heads/5.0.x.zip
+
+      .. note:: App Template
+
+         The ``django-mongodb-app`` template ensures that your ``app.py`` file
+         includes the line ``"default_auto_field = 'django_mongodb_backend.fields.ObjectIdAutoField'"``.
+
+   .. step:: Create models for movie, awards, and viewer data
+
+      Open the ``models.py`` file in the ``sample_mflix`` directory and replace 
+      its contents with the following code:
+
+      .. code-block:: python
+
+         from django.db import models
+         from django_mongodb_backend.fields import EmbeddedModelField, ArrayField
```

Comment:
Note: the EmbeddedModelField import and the `awards` Movie field below will cause errors until [this PR](https://github.com/mongodb-labs/django-mongodb-backend/pull/151) is merged